token-metadata/program/tests/utils/programmable.rs

```diff
@@ -96,8 +152,92 @@ pub async fn create_test_ruleset(
 
     // Client can add additional rules to the Payload but does not need to in this case.
     let payload = Payload::new();
-
     let auth_data = AuthorizationData { payload };
 
     (ruleset_addr, auth_data)
 }
+
```

Comment:
Nitpick: I normally nitpick and ask my team to not have commented out code checked in, but I'm not the authority so feel free to leave it here if you want it for reference.  I just personally prefer to add it back in when its ready to be uncommented/active.